index.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport"
+      content="width=device-width, initial-scale=1">
+    <title>Works</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+   
+    <!-- container-s -->
+    <div class="container sp">
+
+      <!--  wrapper-s -->
+      <div class="wrapper">
+
+        <div class="grid column row">
+          <div class="one"> 
+          </div>
+          <div>Tonic</div>
+          <div>
+            <div>CANOPY</div>
+            <ul class="flex">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div>
+            <ul class="flex">
+              <li>
+                html
+              </li>
```

Comment:
Please try to use the right font and font size of the technologies  provided in the Figma design